spec/controllers/inbox_controller_spec.rb

```diff
@@ -271,19 +271,15 @@
       before { fake_login_known_user(user) }
 
       context "with no comments selected" do
-        it "redirects to inbox with caution and a notice" do
+        it "redirects to the user's inbox with a caution" do
           put :update, params: { user_id: user.login, read: "yeah" }
-          it_redirects_to_with_caution_and_notice(user_inbox_path(user),
-                                                  "Please select something first",
-                                                  "Inbox successfully updated.")
+          it_redirects_to_with_caution(user_inbox_path(user), "Please select something first.")
```

Comment:
Oh, nice. Didn't know `it_redirects_to_with_caution` already included  `expect(flash[:notice]).to be_blank` as a default.